org-brain.el

```diff
@@ -2280,18 +2298,24 @@ Helper function for `org-brain-visualize'."
 (defun org-brain--vis-children (entry)
   "Insert children of ENTRY.
 Helper function for `org-brain-visualize'."
-  (when-let ((children (org-brain-children entry))
-             (fill-col (eval org-brain-child-linebreak-sexp)))
-    (insert "\n\n")
-    (dolist (child (sort children org-brain-visualize-sort-function))
-      (let ((child-title (org-brain-title child))
-            (face (if (member entry (org-brain--local-parent child))
-                      'org-brain-local-child
-                    'org-brain-child)))
-        (when (> (+ (current-column) (length child-title)) fill-col)
-          (insert "\n"))
-        (org-brain-insert-visualize-button child face)
-        (insert "  ")))))
+  (let ((tags (org-brain-get-tags entry t)))
+    (when-let ((children (org-brain-children entry))
+               (fill-col (if (member org-brain-each-child-on-own-line-tag
```

Comment:
Sorry if I seem negative, but I'm not seeing when you'd like to specify for a specific entry that you want the children on one line. Do you have a specific use case in mind?

Reply:
No worries at all.

The case I have in mind is this: I usually wrap children to window width. However, I have a few entries that have children of wildly different sizes. For example, I have an entry for articles. The titles of articles can be very short and very long. The short article names can be quite hard to see because they get buried among these long article names. So for nodes like this, it's nice to see each child on its own line — it makes sure no article gets lost in the list.